diopter/sanitizer.py

```diff
@@ -153,6 +156,7 @@ def __init__(
         *,
         check_warnings: bool = True,
         use_ub_address_sanitizer: bool = True,
+        use_memory_sanitizer: bool = False,
```

Comment:
Should this not be defaulted to `True` ?

Reply:
Is there any reason to disable it by default (other than the extra runtime)

Reply:
Just because for csmith generated program, this is no such issues. And creduce would not introduce this issue according to my experience. So I think it would be faster for your DEAD.